examples/lighting-app/lighting-common/gen/CHIPClustersObjc.h

```diff
@@ -85,6 +85,52 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+/**
+ * Cluster Network Commissioning
+ *
+ */
+@interface CHIPNetworkCommissioning : CHIPCluster
+
+- (void)addThreadNetwork:(char *)operationalDataset
+              breadcrumb:(uint64_t)breadcrumb
+               timeoutMs:(uint32_t)timeoutMs
+       completionHandler:(ResponseHandler)completionHandler;
```

Comment:
is this typical style? 

Maybe at least add a newline in between method declarations. I find this odd to read (but I do not read a lot of objc)

Would some documentation be useful? Like what is a 'breadcrumb'

Reply:
@vivien-apple Could you add a document about how we are using ObjC with CHIP library?

Reply:
I do not have a document handy, but all the view in https://github.com/project-chip/connectedhomeip/tree/master/src/darwin/CHIPTool/CHIPTool/View%20Controllers uses the Objc APIs.

About the documentation itself, we need to fix ZAP to retrieve some documentation from the `xml` definition. At the moment there is no field in the SQL database for it and I believe the JS parser needs to be updated too. @jepenven-silabs  do you have any plans or work in the pipe to add support for that ?